docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -137,9 +139,6 @@ subquestion: |
 
   **IMPORTANT!** If you bring it to the school be sure print a copy for yourself. 
   If you can’t print a copy for yourself, when you bring the letter to the school, ask them to make a copy for you to keep for your records.
-terms:
```

Comment:
Removing this will cause an on screen error where {evaluation} will no longer be defined. Why was this changed? I don't see this requested change from Diana.